### PR TITLE
Moen 20824 geo fence

### PR DESCRIPTION
**Pull request by kkgowtham-engg-sdk**
### Jira Ticket

https://moengagetrial.atlassian.net/browse/MOEN-20824
### Description

Support for Geofence Start/Stop API

### CODE REVIEW

File: geofence/android/settings.gradle [hunk 1]
```diff
@@ -0,0 +1 @@
+rootProject.name = 'moengage_geofence'
```
[the remark this answers is not in the document]
Comment (kkgowtham-engg-sdk) [reply geofence/android/settings.gradle]: Updated

File: geofence/lib/moe_android_geofence.dart [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/services.dart';
+
+import 'moe_geofence_constants.dart';
+import 'moe_geofence_utils.dart';
+
+class MoEAndroidGeofence {
+  String _tag = "${TAG}InAppPayloadMapper";
```
[the remark this answers is not in the document]
Comment (kkgowtham-engg-sdk) [reply geofence/lib/moe_android_geofence.dart]: Fixed

File: geofence/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # MoEngage Geofence Plugin
 
+## Next Release
+- Support for GeoFence Start/Stop API
```
[the remark this answers is not in the document]
Comment (kkgowtham-engg-sdk) [reply geofence/CHANGELOG.md]: Updated

**Review by arshiya-moengage — APPROVED** [review]

**Comment by arshiya-moengage** [comment]
Rebase the branch with development and add logs in geofence dart files

**Review by arshiya-moengage — APPROVED** [review]

**Pull request merged**